### DISCUSSION TITLE
How to remove action log

### DISCUSSION THREAD

**Discussion by geril07** [nvim-tree/nvim-tree.lua#2457] [Q&A] [answered]
When doing some action(create,delete, rename, etc), there is a message which force me to press extra key(enter or smth else). Can i change behaviour, to have only log without extra pressing smth? This behaviour has been added in `85abe29`.
![image](https://github.com/nvim-tree/nvim-tree.lua/assets/62308020/f429b339-08c1-49ad-af6c-462870436a3e)

**Comment by Akmadan23** [comment, marked as answer]
When #2453 will be merged single line messages like this won't need user interaction to be dismissed. However you can always set `notify.threshold` to `vim.log.levels.WARN` so that this kind of messages won't show up again.
